db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, on_delete=models.CASCADE
+    )
+    guild = models.ForeignKey(
+        Guild, on_delete=models.CASCADE, null=True
+    )
+    created_at = models.DateField(auto_now=True)
```

Comment:
1) should be `DateTimeField`
2) Also, `auto_now_add` would be better here. As you had to do this: `a DateTime field, that is set with the current time by default`. The difference is:
- when you use `auto_now_add` - records the time and date of creation
- when you use `auto_now` - updates the value of the field to the current time and date every time when the object changes